recipe/meta.yaml

```
@@ -187,9 +187,9 @@ outputs:
 {% endif %}
     test:
       downstreams:
-        # Temporarily disable dowmstream tests on aarch64 as the solver causes timeouts on drone
-        - cython  # [not aarch64]
-        - setuptools  # [not aarch64]
+        # Temporarily disable downstream test for setuptools due to breaking changes
```

Let's keep the old comment and just add a new one. Since at some point, we presumably want to add these tests back

```suggestion
        # Temporarily disable dowmstream tests on aarch64 as the solver causes timeouts on drone
        # Also temporarily skipping downstream test for setuptools due to breaking changes
```